mfr/extensions/image/export.py

```diff
@@ -30,15 +31,16 @@ def export(self):
                 self.metrics.add('ratio', ratio)
                 if ratio < 1:
                     image = image.resize((round(image.size[0] * ratio), round(image.size[1] * ratio)), Image.ANTIALIAS)
-            if type in ['jpeg', 'jpg']:
-                # handle transparency
-                image = image.convert('RGBA')
-                exported_image = Image.new("RGBA", image.size, (255, 255, 255))
-                exported_image.paste(image, image)
-            else:
-                exported_image = image
-            exported_image.save(self.output_file_path, type)
-            exported_image.close()
+            # handle transparency
+            if image.mode in ('RGBA', 'LA') and type in ['jpeg', 'jpg']:
```

Comment:
This line will trigger for bmp and png images that have transparency in them.
Github issue mention why upgrading Pillow would break our render, and ways to change it (some of which i used) https://github.com/python-pillow/Pillow/issues/2609